starburst-presto/presto.sh

```diff
@@ -123,23 +128,35 @@ function configure_hive(){
 connector.name=hive-hadoop2
 hive.metastore.uri=${metastore_uri}
 EOF
+
+# Add new connectors configs here 
+# For example: Adding jmx properties like
+  cat > presto-server-${PRESTO_VERSION}/etc/catalog/tpch.properties <<- "EOF"
+connector.name=tpch
+tpch.splits-per-node=4
```

Comment:
@mattsfuller do you think we need `tpch.splits-per-node=4`? 
tpch connector defaults to number of CPUs which is a reasonable default.

Reply:
@ChethanUK please remove `tpch.splits-per-node`, let's use defaults.

```suggestion
```

Reply:
I don't think it's needed. I agree with the suggested change.